src/components/index.js

```diff
@@ -1,6 +1,19 @@
-import MovieCardList from "./MovieCardList";
-import MoreButton from "./MoreButton";
-import MovieCard from "./MovieCard";
-import MovieHeader from "./MovieHeader";
-import MovieSearch from "./MovieSearch";
-import SkeletonList from "./SkeletonList";
+import MovieCardList from './MovieCardList';
+import MoreButton from './MoreButton';
+import MovieCard from './MovieCard';
+import MovieHeader from './MovieHeader';
+import MovieSearch from './MovieSearch';
+import SkeletonList from './SkeletonList';
+import MovieModal from './MovieModal';
+import RatingBox from './RatingBox';
+
+export {
+  MovieCardList,
+  MoreButton,
+  MovieCard,
+  MovieHeader,
+  MovieSearch,
+  SkeletonList,
+  MovieModal,
+  RatingBox,
+};
```

Comment:
```suggestion
export { default as MovieCardList } from './MovieCardList';
export { default as MoreButton } from './MoreButton';
export { default as MovieCard } from './MovieCard';
export { default as MovieHeader } from './MovieHeader';
export { default as MovieSearch } from './MovieSearch';
export { default as SkeletonList } from './SkeletonList';
export { default as MovieModal } from './MovieModal';
export { default as RatingBox } from './RatingBox';
```

이렇게 줄일 수 있어요!
또 웹 컴포넌트는 선언한 코드를 읽는 즉시, 즉 런타임에 코드를 평가하면 DOM 요소에 반영되니까 named 부분은 생략되어도 좋습니다!